library/oneview_uplink_set.py

```diff
@@ -93,13 +100,14 @@
     type: complex
 '''
 
+UPLINK_SET_KEY_REQUIRED = "Uplink Set Name and Logical Interconnect required."
 UPLINK_SET_CREATED = 'Uplink Set created successfully.'
 UPLINK_SET_UPDATED = 'Uplink Set updated successfully.'
 UPLINK_SET_DELETED = 'Uplink Set deleted successfully.'
 UPLINK_SET_ALREADY_EXIST = 'Uplink Set already exists.'
 UPLINK_SET_ALREADY_ABSENT = 'Nothing to do.'
-UPLINK_SET_NOT_EXIST = 'Rename failed: Uplink Set not found.'
```

Comment:
I don't get it why you removed the `UPLINK_SET_NOT_EXIST`.  I think, even if the Logical Interconnect is found, the Uplink Set specified may not exist.